### PR TITLE
Open Loop Testing Tips

### PR DESCRIPTION
**Pull request by bpazoles**
Some things we learned and recommend to others while testing the open loop.

### CODE REVIEW

File: docs/docs/walkthrough/phase-3/considerations.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
 # Phase 3: Understanding Your Open Loop
 
 Phase 3 entails understanding what the open loop is recommmending, and why. You should run with an "open loop" for enough days to decde what your max basal setting should be, and also observe how often you disagree or decide to counteract what the loop is recommending.
+
+### Open Loop Testing Phase: Tips & Lessons Learned 
+
+* Use Careportal to it's fullest advantage by entering in every single diabetes event during the times that you're testing the open loop. This will help you get a clearer understanding of what's causing the APS to reccomend any given basal manipulation, and will help you identify areas that need tweaking - whether it be a basal change or formula (ISF, DIA, etc.) change.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-3/considerations.md]: "its"

**Comment by danamlewis** [comment]
Good content, but will you make it as a separate page (still under this phase 3 section)? This will mirror the structure of the rest of the sections. (Your subhead title can be the page title)

**Comment by scottleibrand** [comment]
Great tips. Look forward to seeing this merged ASAP. 😀

**Comment by danamlewis** [comment]
Closing since #245

**Pull request closed without merging**